test/ziggurat/fixtures.clj

```diff
@@ -91,3 +96,37 @@
      (finally
        (delete-queues ~stream-routes)
        (delete-exchanges ~stream-routes))))
+
+(defn mount-producer []
+  (-> (mount/only [#'producer/kafka-producers])
+      (mount/start)))
+
+(defn construct-embedded-kafka-cluster []
+  (doto (EmbeddedKafkaCluster. 1)
+    (.start)))
+
+(def ^:dynamic *embedded-kafka-cluster* nil)
+(def ^:dynamic *bootstrap-servers* nil)
+(def ^:dynamic *consumer-properties* nil)
+(def ^:dynamic *producer-properties* nil)
+
+(defn mount-only-config-and-producer [f]
+  (do
+    (mount-config)
+    (mount-producer)
+    (binding [*embedded-kafka-cluster* (construct-embedded-kafka-cluster)]
+      (binding [*bootstrap-servers* (.bootstrapServers *embedded-kafka-cluster*)]
+        (binding [*consumer-properties* (doto (Properties.)
```

Comment:
would be using one (binding) be sufficient?

Reply:
@macalimlim I tried that. It does not work that way. I could not find a detailed explanation on that at [1] though. There are some references to let, but couldn't understand it.

[1] https://clojuredocs.org/clojure.core/binding